src/components/source_editor/LatticeConfigurationDialog.js

```diff
@@ -150,7 +179,8 @@ class LatticeConfigurationDialog extends React.Component {
                                 <label className="fg-label">Lattice 'a'</label>
                                 <div className="fg-line">
                                     <input type="number" name="a" className="form-control fc-alt fg-input" min="0"
-                                        step="0.05" disabled={this.isDisabled('a')}
```

Comment:
function this.isDisabled(param) (see line 67) always returns false in current state